docs/guide.rst

```diff
@@ -185,11 +185,10 @@ the scene as desired::
         gfx.MeshPhongMaterial(color="#336699"),
     )
 
+    rot = la.quaternion_make_from_euler_angles((0, 0.01), order="XY")
```

Comment:
Not for this PR. I am still in favor of this change, and would also like to see `vector_euler_angles_from_quaternion` and/or `vector_euler_angles_from_rotation_matrix` so that we can have `obj.world.rotation_{x,y,z}` properties. At the same time, this is a change that is independent of the refactor of pygfx so I would make it a separate PR.

Reply:
> so that we can have `obj.world.rotation_{x,y,z}` properties

Amazing! :)